web_app_deployment/roles/kubectl/templates/deploy_web_app.yml.j2

```diff
@@ -50,3 +50,78 @@ spec:
               serviceName: piezo-app-service
               servicePort: web
             path: /piezo/
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    kubernetes.io/ingress.class: nginx
+  name: prometheus-ingress
+spec:
+  rules:
+    - host: {{ k8s_master_url }}  # Node where ingress controller is run (kuberenetes master)
+      http:
+        paths:
+          - backend:
+              serviceName: piezo-monitor-prometheus-o-prometheus
```

Comment:
Can we reconsider this service name?

Reply:
I'm afraid this is automatically named by the prometheus operator. The `piezo-monitor` part is set when prometheus is installed with helm `helm install stable/prometheus-operator --name piezo-monitor --set prometheus.prometheusSpec.routePrefix=/prometheus/ --set prometheus.prometheusSpec.externalUrl={k8s master url}:{ingress port}/prometheus/` but the rest is added by the operator. Here we are just specifying that we want to listen on this service and scrape the metrics from it